spec.html

```diff
@@ -943,6 +946,9 @@ <h1>The Undefined Type</h1>
     <emu-clause id="sec-ecmascript-language-types-null-type">
       <h1>The Null Type</h1>
       <p>The Null type has exactly one value, called *null*.</p>
+      <emu-note>
+        <p>Value *null* used as a root of prototype chain if there is no inheritance. Keep in mind *typeof* operator also returns "object" for *null* value: <emu-xref href="#sec-typeof-operator"></emu-xref>. A careless combination of these facts can be dangerous.</p>
```

Comment:
Words like "careless" and "dangerous" do not belong in a specification, and there's no benefit in restating how `typeof` works, since it's already defined in this document.